Cargo.lock

```diff
@@ -391,6 +391,7 @@ version = "0.1.0"
 dependencies = [
  "base64",
  "clap",
+ "eif_defs",
```

Comment:
why is this needed now ?

Reply:
Because `enclave_build` now depends on `eif_defs` and `Cargo.lock` required update.